packages/contracts/tests/js/StabilityPoolTest.js

```diff
@@ -1298,6 +1302,70 @@ contract('StabilityPool', async accounts => {
+      await openTrove({ extraZUSDAmount: toBN(dec(200, 18)), ICR: toBN(dec(2, 18)), extraParams: { from: B } });
+      await openTrove({ extraZUSDAmount: toBN(dec(300, 18)), ICR: toBN(dec(2, 18)), extraParams: { from: C } });
+
+      // A, B, C, provide to SP
+      await stabilityPool.provideToSP(dec(10, 18), frontEnd_1, { from: A });
+      await stabilityPool.provideToSP(dec(20, 18), frontEnd_2, { from: B });
+      await stabilityPool.provideToSP(dec(30, 18), ZERO_ADDRESS, { from: C });
+
+      await th.fastForwardTime(timeValues.SECONDS_IN_ONE_HOUR, web3.currentProvider);
+
+      // Get A, B, C ZERO balance before
+      const A_ZEROBalance_Before = await zeroToken.balanceOf(A);
+      const B_ZEROBalance_Before = await zeroToken.balanceOf(B);
+      const C_ZEROBalance_Before = await zeroToken.balanceOf(C);
+
+      // A, B, C top up
+      await stabilityPool.provideToSP(dec(10, 18), frontEnd_1, { from: A });
+      await stabilityPool.provideToSP(dec(20, 18), frontEnd_2, { from: B });
+      await stabilityPool.provideToSP(dec(30, 18), ZERO_ADDRESS, { from: C });
+
+      // Get ZERO balance after
+      const A_ZEROBalance_After = await zeroToken.balanceOf(A);
+      const B_ZEROBalance_After = await zeroToken.balanceOf(B);
+      const C_ZEROBalance_After = await zeroToken.balanceOf(C);
+
+      // Check ZERO Balance of A, B, C hasn't increased as there were no Zero tokens
+      assert.isTrue(A_ZEROBalance_After.eq(A_ZEROBalance_Before));
+      assert.isTrue(B_ZEROBalance_After.eq(B_ZEROBalance_Before));
+      assert.isTrue(C_ZEROBalance_After.eq(C_ZEROBalance_Before));
+
+      // await communityIssuance.receiveZero(owner, toBN(dec(30, 24)));
+
+      // A, B, C top up again
+      await stabilityPool.provideToSP(dec(10, 18), frontEnd_1, { from: A });
+      await stabilityPool.provideToSP(dec(20, 18), frontEnd_2, { from: B });
+      await stabilityPool.provideToSP(dec(30, 18), ZERO_ADDRESS, { from: C });
+
+      await th.fastForwardTime(timeValues.SECONDS_IN_ONE_HOUR, web3.currentProvider);
+
+      // A, B, C top up again
+      await stabilityPool.provideToSP(dec(10, 18), frontEnd_1, { from: A });
+      await stabilityPool.provideToSP(dec(20, 18), frontEnd_2, { from: B });
+      await stabilityPool.provideToSP(dec(30, 18), ZERO_ADDRESS, { from: C });
+
+      // Get ZERO balance after to trigger reward payments
+      const A_ZEROBalance_AfterReceiving = await zeroToken.balanceOf(A);
+      const B_ZEROBalance_AfterReceiving = await zeroToken.balanceOf(B);
+      const C_ZEROBalance_AfterReceiving = await zeroToken.balanceOf(C);
+
+      // Gains should be sent
+      /* disabled as zero token is not used in beta 
+      close presale right after the deployment
+      assert.isTrue(A_ZEROBalance_AfterReceiving.gt(A_ZEROBalance_Before))
+      assert.isTrue(B_ZEROBalance_AfterReceiving.gt(B_ZEROBalance_Before))
+      assert.isTrue(C_ZEROBalance_AfterReceiving.gt(C_ZEROBalance_Before))
+      */
```

Comment:
why is the relevant part commented out?

Reply:
It was commented out since the beginning (from another branch).
So i don't think we need to pull out the comment here?